perun-openapi/openapi.yml

```diff
@@ -202,6 +202,27 @@ components:
       discriminator:
         propertyName: beanName
 
+    ActionType:
+      type: string
+      description: "Action types for attributes."
+      enum:
+        - "WRITE"
+        - "WRITE_VO"
+        - "WRITE_PUBLIC"
+        - "READ"
+        - "READ_VO"
+        - "READ_PUBLIC"
+
+    AttributeRights:
+      allOf:
```

Comment:
allOf is not needed when the type does not inherit from another type

Reply:
Fixed.